packages/app/src/views/Home.vue

```diff
@@ -1,7 +1,8 @@
 <template>
-  <div id="wrapper" class="mx-auto max-w-lg h-full w-full flex flex-col">
+  <div class="app mx-auto max-w-lg h-full w-full flex flex-col">
```

Comment:
why has the home component the app class? It's quite confusing when the view does not implement the style which is given in the class.

Reply:
Öhhm. Just wanted to get rid of the id as an id has to be unique in html and `wrapper` is quite generic.

Reply:
resolved by e66b6422db31be3a0e73d259933e7c13f5a4d5a7